queries.sql

```diff
@@ -7,4 +7,64 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3 OR weight_kg IN (10.4, 17.3);
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3 OR weight_kg IN (10.4, 17.3);
+
+/* Transaction */
+BEGIN;
+
+UPDATE animals SET species = 'unspecified';
+
+ROLLBACK;
+
+/* Another Transaction */
+
+BEGIN;
+
+UPDATE animals SET species = 'digimon' WHERE RIGHT(name, 3) = 'mon';
+UPDATE animals SET species = 'pokemon' WHERE species = 'null';
+
+COMMIT;
+
+SELECT * FROM animals
+
+/* Another Transaction */
+
+BEGIN;
+
+DELETE FROM animals;
+
+ROLLBACK;
+
+SELECT * FROM animals
+
```

Comment:
- [x] You have created a query great but it would be nice if you would verify your changes before and after Rollback 👍
Your query will look something like this ⬇️
```sql
   BEGIN;

   DELETE FROM animals;
   SELECT COUNT(*) FROM ANIMALS;

   ROLLBACK;
   SELECT COUNT(*) FROM ANIMALS;
```